tests/orquestra/vqa/algorithms/vqe_test.py

```diff
@@ -0,0 +1,185 @@
+import numpy as np
+import pytest
+from orquestra.opt.optimizers import ScipyOptimizer
+from orquestra.quantum.api.estimation import EstimationPreprocessor
+from orquestra.quantum.backends.symbolic_simulator import SymbolicSimulator
+from orquestra.quantum.operators import PauliTerm
+
+from orquestra.vqa.algorithms.vqe import VQE
+from orquestra.vqa.ansatz.quantum_compiling import HEAQuantumCompilingAnsatz
+from orquestra.vqa.estimation.cvar import CvarEstimator
+from orquestra.vqa.grouping import group_greedily, group_individually
+from orquestra.vqa.shot_allocation import (
+    allocate_shots_proportionally,
+    allocate_shots_uniformly,
+)
+
+
+@pytest.fixture()
+def hamiltonian():
+    return PauliTerm("Z0") + PauliTerm("Z1")
+
+
+@pytest.fixture()
+def optimizer():
+    return ScipyOptimizer(method="L-BFGS-B")
+
+
+@pytest.fixture()
+def backend():
+    return SymbolicSimulator()
+
+
+@pytest.fixture()
+def ansatz(hamiltonian):
+    return HEAQuantumCompilingAnsatz(1, hamiltonian.n_qubits)
+
+
+@pytest.fixture()
+def initial_params(hamiltonian):
+    return np.random.random(hamiltonian.n_qubits)
+
+
+@pytest.fixture()
+def vqe_object(hamiltonian, ansatz):
+    return VQE.default(hamiltonian=hamiltonian, ansatz=ansatz)
+
+
+class TestDefaultVQE:
+    def test_default_optimizer_is_lbfgsb(self, vqe_object):
+        assert vqe_object.optimizer.method == "L-BFGS-B"
+
+    def test_default_estimation_init_default_value(self, vqe_object):
```

Comment:
Rename to `test_default_estimation_method_is_...`? 
Same in the following.
I think you followed a pattern from a slightly outdated version of QAOA, check out how it's done on `main` now (I might be wrong though!)

Reply:
Its the same in the main

Reply:
Well..
Could you refactor it in both QAOA and VQA to have better naming then 😅 ?